src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/adapters/TreeTableAdapter.js

```diff
@@ -134,6 +157,23 @@ class TreeTableAdapter extends AbstractTableAdapter {
                 </Table.Body>
             </Table>
         );
+
+        if (!pagination || pageCount === undefined || data.length === 0) {
```

Comment:
would like to use the same condition in all adapter. the `TableAdapter` condition is battle-tested 🙂